docs/main/Solutioning/backup-and-restore/velero-br-fs.md

```diff
@@ -0,0 +1,316 @@
+---
+id: velerobrfs
+title: Velero Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+keywords: 
+  - Velero Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+  - Velero Backup and Restore
+  - FileSystem
+description: In this document, you learn about Velero Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem.
+---
+
+# Velero Backup and Restore using Replicated PV Mayastor Snapshots - FileSystem
+
+Using Velero for backup and restore operations with Replicated PV Mayastor snapshots combines the strengths of both tools, providing a robust, efficient, and easy-to-manage solution for protecting your Kubernetes applications and data. The integration leverages the high performance and efficiency of Replicated PV Mayastor snapshots with the comprehensive backup capabilities of Velero, ensuring your data is always protected and recoverable.
+
+Velero can support any volume provider that has a Container Storage Interface (CSI) driver with snapshotting capability.
+
+## Requirements
+
+### Replicated PV Mayastor
+
+Replicated PV Mayastor CSI supports volume snapshots, which are essential for backup and recovery operations. By using Velero with Replicated PV Mayastor CSI, you can take advantage of Velero's robust backup and restore capabilities to create point-in-time snapshots of your volumes. This is critical for disaster recovery and data protection strategies.
+
+Make sure you have installed Replicated PV Mayastor before proceeding to the next step. Refer to the [OpenEBS Installation documentation](../../quickstart-guide/installation.md#installation-via-helm) to install Replicated PV Mayastor using Helm.
+
+### Velero with CSI Snapshot Support
+
+Velero is an open-source tool used for backup, restore, and migration of Kubernetes cluster resources and persistent volumes. It allows users to schedule regular backups of their Kubernetes resources and persistent volumes, and to restore those resources when needed.
+
+When CSI snapshots are enabled, Velero leverages the Kubernetes CSI driver to create snapshots of persistent volumes. These snapshots can be used to restore data to the same or different Kubernetes clusters.
+
+## Details of Setup
+
+### Install Velero with CSI Snapshot Support Enabled
+
+Install Velero (V1.13.2) with the `velero-plugin-for-csi` and with any S3 object storage. In this example, we are using the Google cloud bucket as our BackupStorageLocation. Refer to [Velero Documentation](https://velero.io/plugins/) for instructions on setting up the BackupStorageLocation.
+
+As an example we will be using Google Cloud Platform (GCP) and OpenEBS Plugin.
```

Comment:
We can remove the "plugin" here. We are not using plugin here. we have removed this from command must be a typo here.

As an example we will be using Google Cloud Platform (GCP) and OpenEBS